types/index.d.ts

```diff
@@ -1,21 +1,28 @@
-import { Moment, MomentFormatSpecification, MomentInput } from 'moment'
+import moment from 'moment'
 import Vue from 'vue'
 
 declare module '@nuxt/vue-app' {
   interface Context {
-    $moment(input?: MomentInput, format?: MomentFormatSpecification, language?: string, strict?: boolean): Moment
+    $moment: typeof moment
   }
 }
 
 // Nuxt 2.9+
 declare module '@nuxt/types' {
   interface Context {
-    $moment(input?: MomentInput, format?: MomentFormatSpecification, language?: string, strict?: boolean): Moment
+    $moment: typeof moment
   }
 }
 
 declare module 'vue/types/vue' {
   interface Vue {
-    $moment(input?: MomentInput, format?: MomentFormatSpecification, language?: string, strict?: boolean): Moment
+    $moment: typeof moment
   }
 }
+
+declare module 'vuex' {
```

Comment:
Use `vuex/types/index` instead of `vuex`

See https://github.com/nuxt-community/axios-module/commit/9205fdf0381eff052388331c13ff774666627bc5

Reply:
Fixed in 34b0a76860d747db3de899ec930bdb87d0ba4b8a